recipes/debug_assert/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(test_package)
+
+set(CMAKE_VERBOSE_MAKEFILE TRUE)
+set(CMAKE_CXX_STANDARD 11)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
```

Comment:
```suggestion
target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
```

Reply:
To be honest, I copied the test_package recipe from one of the existing packages (With no modifications) to be coherent with the rest of the recipes, so...

Reply:
I used this one https://github.com/conan-io/conan-center-index/blob/master/recipes/stb/all/test_package/CMakeLists.txt

But I see most packages follow your recomendation so I will change it :)